compiler/env/OMRCPU.hpp

```diff
@@ -125,12 +125,12 @@ class OMR_EXTENSIBLE CPU
    // Processor identity and generation comparisons
    //
    TR_Processor id() { return _processor; }
-   bool is(TR_Processor p) { return _processor == p; }
+   bool is_old(TR_Processor p) { return _processor == p; }
    bool isNot(TR_Processor p) { return _processor != p; }
-   bool isAtLeast(TR_Processor p) { return _processor >= p; }
+   bool isAtLeast_old(TR_Processor p) { return _processor >= p; }
```

Comment:
What are these `_old` APIs supposed to represent?

Reply:
I think it'll be better to have the x86 PR(https://github.com/eclipse/omr/pull/5151) merged first. These old queries `bool isAtLeast(TR_Processor p)` clashes with my new queries `bool isAtLeast(OMRProcessorDesc)` on p and z so I just temporarily changed it to `isAtLeast_old`. (There are 2 instances where the old api was used). Well, I already got rid of them in https://github.com/eclipse/omr/pull/5151.

Reply:
old api being `bool isAtLeast(TR_Processor p)` and new api being `bool isAtLeast(OMRProcessorDesc)`. xlC on p and z doesn't seem to distinguish enum types so they clash if the function name is the same.

Reply:
Actually I'll just remote these APIs. None of them is actually used.